src/redux/actionTypes.ts

```diff
@@ -33,6 +33,9 @@ export enum ActionTypes {
   COLONY_EXTENSION_UPGRADE = 'COLONY_EXTENSION_UPGRADE',
   COLONY_EXTENSION_UPGRADE_ERROR = 'COLONY_EXTENSION_UPGRADE_ERROR',
   COLONY_EXTENSION_UPGRADE_SUCCESS = 'COLONY_EXTENSION_UPGRADE_SUCCESS',
+  COLONY_EXTENSION_UPLOAD_ADDRESSES = 'COLONY_EXTENSION_UPLOAD_ADDRESSES',
+  COLONY_EXTENSION_UPLOAD_ADDRESSES_ERROR = 'COLONY_EXTENSION_UPLOAD_ADDRESSES_ERROR',
+  COLONY_EXTENSION_UPLOAD_ADDRESSES_SUCCESS = 'COLONY_EXTENSION_UPLOAD_ADDRESSES_SUCCESS',
```

Comment:
I would have only added these actions, when the saga gets added, otherwise, the person who implements the saga, might not know about them and add them a second time, with a different name _(and we don't catch that in the review)_

Reply:
I added them, because ActionForm was screaming on me